ci-cd-bootstrap/inventory/group_vars/all.yml

```diff
@@ -26,8 +26,16 @@ openshift_cluster_content:
     namespace: labs-ci-cd
   - name: sonarqube
     template: "{{ inventory_dir }}/../../templates/sonarqube/template.json"
-    params: "{{ inventory_dir }}/../params/sonarqube/build_params"
+    params: "{{ inventory_dir }}/../params/sonarqube/build-and-deploy"
     namespace: labs-ci-cd
+    tags:
+    - sonar
+  - name: sonarqube-postgresql
+    template: "openshift//postgresql-persistent"
+    params: "{{ inventory_dir }}/../params/sonarqube/postgresql"
+    namespace: labs-ci-cd
+    tags:
```

Comment:
The only feedback I have is that we need to remember to add the post actions for setting the labels once the openshift-applier has been updated with that capability

Reply:
good call: https://github.com/rht-labs/labs-ci-cd/issues/46